data/pc/common/features.json

```diff
@@ -664,6 +664,14 @@
     "description": "allows for usage of `signature` instead of verifyToken in serverbound packet_encryption_begin",
     "versions": ["1.19_major", "latest"]
   },
+  {
+    "name": "signatureV2",
```

Comment:
try "chatSignatureV2" to be less ambiguous

Reply:
changed to `profileKeySignatureV2` to specify this is the format used when signing key during initial handshake